<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qpid::AudioNavigateDialog</class>
 <widget class="QDialog" name="Qpid::AudioNavigateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QpidPhone Login</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background: #3e8bc2;
	border: 1px solid #3d89bf;
}

#frameCaption {
	background: rgb(49,169,221);
}

/* Caption style sheet */
#frameCaption {
	border-image: url(:/images/ui/top_caption.png);
}

#pushButtonMinimize {
	border-image: url(:/images/ui/minimize.png);
}

#pushButtonMinimize:hover {
	border-image: url(:/images/ui/minimize_hover.png);
}

#pushButtonClose {
	border-image: url(:/images/ui/close.png);
}

#pushButtonClose:hover {
	border-image: url(:/images/ui/close_hover.png);
}

#labelTitle {
	color: rgb(255,255,255);
	padding-left: 3px;
	font-size: 15px;
	font-weight: bold;
}

/* End of caption style sheet */

#labelHeadphone {
	border-image: url(:/images/ui/step_left_selected.png);
}
#labelMiddle {
	border-image: url(:/images/ui/step_play_selected.png);
}
#labelMicrophone {
	border-image: url(:/images/ui/step_right_unselect.png);
}

#frameTop,#frameBottom,#pagePlayback,#pageCapture {
	background: #ebf4fa;
	border: 1px solid #3d89bf;
}

#frameBottom {
	border-bottom-left-radius: 5px;
	border-bottom-right-radius: 5px;
}

/* Welcome style sheet */

#labelWelcome {
	color: #333333;
	font-size: 22px;
	font-weight: bold;
	font-style: italic;
}

#labelDesp {
	color: #333333;
	font-size: 14px;
}

/* Tab style sheet */
#labelTabTitle{
	font-size: 22px;
	color: #ffffff;
	font-weight: bold;
	font-style: italic;
}

#labelTabDesp {
	font-size: 16px;
	color: #ffffff;
	font-weight: bold;
}

#labelHeadphone,#labelMicrophone {
	color: #4c801a;
	font-size: 18px;
	font-weight: bold;
}

/* */
#labelPlayback,#labelCapture {
	font-size: 14px;
}

#pushButtonPlaybackFix,#pushButtonPlaybackNext,#pushButtonPlaybackSkip,#pushButtonCaptureFix,#pushButtonCaptureFinish {
	border: 0px;
	background-image: url(:/images/ui/button_bg.png);
	min-width: 130px;
	max-width: 130px;
	min-height: 30px;
	max-height: 30px;
}

#pushButtonPlayback {
	border-image: url(:/images/ui/play_normal.png);
}
#pushButtonPlayback:pressed {
	border-image: url(:/images/ui/play_pressed.png);
}
#pushButtonPlayback:disabled {
	border-image: url(:/images/ui/play_disabled.png);
}
#pushButtonCapture {
	border-image: url(:/images/ui/record_normal.png);
}
#pushButtonCapture:pressed {
	border-image: url(:/images/ui/record_pressed.png);
}
#pushButtonCapture:disabled {
	border-image: url(:/images/ui/record_disabled.png);
}

/* ComboBox */
QComboBox {
	border: 1px solid #cccccc;
	border-radius: 3px;
	min-height: 35px;
	padding-left: 10px;
}

QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 40px;
	border-top-right-radius: 3px; /* same radius as the QComboBox */
	border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
	image: url(:/images/ui/input_bg.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
	top: 1px;
	left: 1px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameCaption">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>Voice Easy</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="sysPanelContainer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="sysPanel" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButtonMinimize">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonClose">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameTop">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="labelWelcome">
        <property name="text">
         <string>Welcome to Voice Easy</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>430</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelDesp">
        <property name="text">
         <string>We connect the world</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameTabPanel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>180</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTabTitle">
        <property name="text">
         <string>You are nearly there</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="tabContainer" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelHeadphone">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Test your headphone</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMiddle">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMicrophone">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Test your microphone</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>424</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTabDesp">
        <property name="text">
         <string>Test your audio device</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="deviceContainer">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pagePlayback">
      <widget class="QLabel" name="labelPlayback">
       <property name="geometry">
        <rect>
         <x>39</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Playback devices:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboPlayback">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>17</y>
         <width>251</width>
         <height>37</height>
        </rect>
       </property>
      </widget>
      <widget class="QFrame" name="playbackPanel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>641</width>
         <height>81</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="playbackDesp">
          <property name="text">
           <string>Click play button to play sound</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="playbackNext" native="true">
          <property name="minimumSize">
           <size>
            <width>460</width>
            <height>0</height>
           </size>
          </property>
          <widget class="QLabel" name="labelPlaybackQues">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>5</y>
             <width>241</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Did you hear the sound?</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonPlaybackFix">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>130</width>
             <height>30</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>No,how to fix ?</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonPlaybackNext">
           <property name="geometry">
            <rect>
             <x>145</x>
             <y>30</y>
             <width>130</width>
             <height>30</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Yes,next</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonPlaybackSkip">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>30</y>
             <width>130</width>
             <height>30</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Skip</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButtonPlayback">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>65</y>
         <width>73</width>
         <height>73</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>73</width>
         <height>73</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>73</width>
         <height>73</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pageCapture">
      <widget class="QLabel" name="labelCapture">
       <property name="geometry">
        <rect>
         <x>39</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Capture devices:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboCapture">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>17</y>
         <width>251</width>
         <height>37</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonCapture">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>65</y>
         <width>73</width>
         <height>73</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>73</width>
         <height>73</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>73</width>
         <height>73</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QFrame" name="capturePanel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>591</width>
         <height>81</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>73</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="captureDesp">
          <property name="text">
           <string>Hold left-click on the button to speak</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="captureNext" native="true">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>0</height>
           </size>
          </property>
          <widget class="QLabel" name="labelCaptureQues">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>321</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Did it play your voice when you released the button ?</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonCaptureFix">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>30</y>
             <width>130</width>
             <height>30</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>No,how to fix ?</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonCaptureFinish">
           <property name="geometry">
            <rect>
             <x>143</x>
             <y>30</y>
             <width>130</width>
             <height>30</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Yes,finish</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="labelMovie">
       <property name="geometry">
        <rect>
         <x>113</x>
         <y>100</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frameBottom">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="phone.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>Qpid::AudioNavigateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMinimize</sender>
   <signal>clicked()</signal>
   <receiver>Qpid::AudioNavigateDialog</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>loginStarted()</signal>
  <signal>loginSucceeded()</signal>
  <signal>loginFailed()</signal>
  <slot>login()</slot>
 </slots>
</ui>
